src/node/build/index.ts

```diff
@@ -679,22 +679,22 @@ export async function ssrBuild(
     outDir: 'dist-ssr',
     ...options,
     rollupPluginVueOptions: {
-      ...rollupPluginVueOptions,
-      target: 'node'
+      target: 'node',
+      ...rollupPluginVueOptions
     },
     rollupInputOptions: {
-      ...rollupInputOptions,
       external: resolveExternal(
         rollupInputOptions && rollupInputOptions.external
-      )
+      ),
+      ...rollupInputOptions
```

Comment:
move this one back to the top

Reply:
@aleclarson Done. Perhaps would be interesting using something like [merge-options](https://github.com/schnittstabil/merge-options) at some point?